README.md

```diff
@@ -158,3 +158,21 @@ Example:
 - Updates config bonfire file and add rbac component
 - Deploys rbac together with relationships application
   - Hardcoded image is used with grpc client for calling relationships
+
+### Running IQE Tests
+
+It is possible to run the [IQE Relations API Plugin](https://gitlab.cee.redhat.com/insights-qe/iqe-relations-api-plugin) tests against the service using the `bonfire deploy-iqe-cji` command.
```

Comment:
You cannot have internal repo liinks in public Github repos. You need to remove this. There is Kessel internal docs repo for RH employees only, you need to add the repo link there. Probably since IQE is downstream, you can move the entire section of this to internal repo.

Reply:
Thank you for the warning.
Going to update the internal docs.